addons/web/static/src/views/widgets/ribbon/ribbon.scss

```diff
@@ -6,6 +6,7 @@
     height: var(--Ribbon-wrapper-height);
     overflow: hidden;
     position: absolute;
+    z-index: 0;
```

Comment:
I don't think it is appropriate. You should check what is wrong with --Ribbon-z-index